### PR TITLE
Add docs for editor viewport drawing methods

### PR DESCRIPTION
**Pull request by KoBeWi**
I was making a plugin and this was missing, so I thought I'd document it after having figured it out.

### CODE REVIEW

File: doc/classes/EditorPlugin.xml [hunk 1]
```diff
@@ -210,6 +210,17 @@
 			<argument index="0" name="overlay" type="Control">
 			</argument>
 			<description>
+				Called by the engine when the 2D editor's viewport is updated. Use the [code]overlay[/code] [Control] for drawing. You can update the viewport manually by calling [method update_overlays].
+				[codeblock]
+				func forward_canvas_draw_over_viewport(overlay):
+					#draw a circle at cursor position
```
Comment (Xrayez) [inline doc/classes/EditorPlugin.xml:216]: Do we use the same comments style in GDScript as in C++?
```suggestion
					# Draw a circle at cursor position.
```

File: doc/classes/EditorPlugin.xml [hunk 2]
```diff
@@ -210,6 +210,17 @@
...
+					overlay.draw_circle(overlay.get_local_mouse_position(), 64)
+				
+				func forward_canvas_gui_input(event):
+					if event is InputEventMouseMotion:
+						#redraw viewport when cursor is moved
```
Comment (Xrayez) [inline doc/classes/EditorPlugin.xml:221]:
```suggestion
						# Redraw viewport when cursor is moved.
```

**Comment by akien-mga** [comment]
`cd doc && make rst` is your friend ;)

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by HaSa1002** [comment]
Ah well. Will create a PR soon to port the code over :)

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.4.

**Pull request merged**